src/components/Sidebar/Sidebar.js

```diff
@@ -125,45 +136,80 @@ class Sidebar extends PureComponent<Props, State> {
         {({ sidebarOffsetPercentage, firstProjectPosition }) => (
           <Fragment>
             <Wrapper offset={`${sidebarOffsetPercentage}%`}>
-              <ScrollbarOnlyVertical
-                autoHide
-                renderTrackHorizontal={props => (
-                  <div {...props} style={{ display: 'none' }} />
-                )}
-              >
-                <IntroductionBlurb
-                  isVisible={!finishedOnboarding && introSequenceStepIndex >= 1}
-                />
-
-                <Projects offset={`${firstProjectPosition}px`}>
-                  {projects.map(project => (
-                    <Fragment key={project.id}>
-                      <Tooltip title={project.name} position="right">
-                        <SidebarProjectIcon
-                          size={SIDEBAR_ICON_SIZE}
-                          id={project.id}
-                          name={project.name}
-                          color={project.color}
-                          iconSrc={project.icon}
-                          isSelected={
-                            finishedOnboarding &&
-                            project.id === selectedProjectId
+              <DragDropContext onDragEnd={this.onDragEnd}>
+                <ScrollbarOnlyVertical
+                  autoHide
+                  renderTrackHorizontal={props => (
+                    <div {...props} style={{ display: 'none' }} />
+                  )}
+                >
+                  <Droppable droppableId="droppable">
+                    {provided => (
+                      <div ref={provided.innerRef}>
+                        <IntroductionBlurb
```

Comment:
@beyersito The vertical scrolling issue is caused by the `IntroductionBlurb` component. It's hidden but uses the space. That was a bit tricky to find. The blurb is displayed after the first project was created.

By rendering it conditionally you can fix this:
```js
{
  !finishedOnboarding &&
    introSequenceStepIndex >= 1 && ( 
       <IntroductionBlurb isVisible = {!finishedOnboarding &&
          introSequenceStepIndex >= 1}/>
    )
}
```
Maybe you can also move the `isVisible` condition to a constant so the conditional looks better. Like `const displayIntroductionBlurb = !finishedOnboarding && introSequenceStepIndex >= 1`

Please also remove `isVisible` prop from the `IntroductionBlurb` component as it's not needed anymore with the conditional rendering. I kept in the snippet above so you can test it as it is.

With this you can also remove the code with the vertical scrollbar where we're using `display:none` from line 142. Because the scrollbar won't be rendered.

Reply:
@beyersito Even better than conditional rendering: You can move the `IntroductionBlurb` outside of the scrollbars component. That way you're also not getting the scroll-x issue and you can leave the `IntroductionBlurb` unmodified.

I've also noticed that the IntroductionBlurb is not displayed on master.